india_compliance/gst_india/utils/gstin_info.py

```diff
@@ -49,6 +54,42 @@ def get_gstin_info(gstin):
     return gstin_info
 
 
+def get_gstin_log(gstin):
+    archive_party_information = frappe.get_cached_value(
+        "GST Settings", None, "archive_party_information"
+    )
+    current_date = frappe.utils.now_datetime()
+    archive_date = current_date - timedelta(days=archive_party_information)
+
+    integration_log_output = frappe.get_all(
+        "Integration Request",
+        {
+            "status": "Completed",
+            "url": ("like", "%" + "search"),
+            "data": ("like", "%" + gstin + "%"),
+            "creation": ["between", (archive_date, current_date)],
```

Comment:
![image](https://github.com/resilient-tech/india-compliance/assets/10496564/310dfc73-4c8f-4767-930c-b3f65174955c)

Use modified as its value is indexed.